internal/packages.bzl

```diff
@@ -0,0 +1,52 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Rules to install NodeJS dependencies during WORKSPACE evaluation."""
+
+def _npm_install_impl(repository_ctx):
+  """Core implementation of npm_install."""
+
+  repository_ctx.file("BUILD", """
+package(default_visibility = ["//visibility:public"])
+filegroup(name = "node_modules", srcs = glob(["node_modules/**/*"]))
```

Comment:
there's a problem: many npm packages ship with files that are not valid labels. sometimes the user currently needs to give specific glob patterns under node_modules which are safe, eg. https://github.com/angular/angular/blob/master/BUILD.bazel#L7

If that's not possible with bazel-managed-deps, could you at least document that in the tradeoffs section in the readme?